lib/exception_handling/exception_info.rb

```diff
@@ -249,6 +249,7 @@ def dump_hash( h, indent_level = 0 )
     def enhanced_data_to_honeybadger_context
       data = enhanced_data.dup
       data[:server] = ExceptionHandling.server_name
+      data[:context] = @exception_context if @exception_context.present?
```

Comment:
are we going to regret it being "context > context" in honeybadger?  I know that's what we had said before, just my only concern being confusion around the two layers there. Maybe "extra_context" or "invoca_context" ?